kernel/src/main/java/org/kframework/kompile/KompileOptions.java

```diff
@@ -135,6 +137,9 @@ public boolean isKore() {
         @Parameter(names="--gen-bison-parser", description="Emit bison parser for the PGM configuration variable within the syntax module of your definition into the kompiled definition.")
         public boolean genBisonParser;
 
+        @Parameter(names="--gen-glr-bison-parser", description="Emit GLR bison parser for the PGM configuration variable within the syntax module of your definition into the kompiled definition.")
+        public boolean genGlrBisonParser;
```

Comment:
We should emit a parser for each `$` variable that shows up in the configuration, not just `$PGM`.